### PR TITLE
skeleton for SDK and Fyle Projects

### PR DESCRIPTION
**Pull request by vishalsoni242**

### CODE REVIEW

File: src/apis/api_base.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        this.access_token = access_token;
+    }
+
+    set_server_url(server_url) {
+        this.server_url = server_url;
+    }
+
+    async get_request(params, api_url) {
+        Object.keys(params).forEach(
+            key => params[key] === undefined && delete params[key]
```
Comment (snarayanank2) [inline src/apis/api_base.js]: split into multiple lines - this is difficult to grok

File: src/apis/api_base.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        );
+
+        Object.keys(params).forEach(
+            key =>
+                typeof params[key] === 'boolean' &&
```
Comment (snarayanank2) [inline src/apis/api_base.js]: same here.. split into multi-line function

Comment (Shwetabhk) [reply src/apis/api_base.js]: +1

File: src/apis/api_base.js (hunk 1 above)
Comment (Shwetabhk) [reply src/apis/api_base.js]: +1

File: src/apis/api_base.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+        let response = await fetch(url, {
+            method: 'GET',
+            headers: {
+                Authorization: `Bearer ${this.access_token}`,
+                'Content-type': 'application/json'
+            }
+        });
+        let data = await response.json();
+        if (response.status == 200) {
```
Comment (Shwetabhk) [inline src/apis/api_base.js]: I think it would be better to use `===` everywhere

File: package.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "fyle-sdk-node",
+    "version": "1.0.0",
```
Comment (Shwetabhk) [inline package.json]: Please start the version from 0.1.0

File: src/apis/projects.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+const ApiBase = require('./api_base');
+
+module.exports = class Projects extends ApiBase {
+    constructor() {
+        super();
```
Comment (Shwetabhk) [inline src/apis/projects.js]: @vishalsoni242 can we add docstrings everywhere, like in Python SDK? It is better to add it off as you proceed.

**Pull request merged**